plugins/woocommerce-blocks/package.json

```diff
@@ -359,12 +359,13 @@
 					"name": "JavaScript",
 					"command": "test:js",
 					"changes": [
-						"jest.config.js",
 						"webpack.config.js",
 						"babel.config.js",
 						"tsconfig.json",
 						"assets/**/*.{js,ts,tsx,scss}",
-						"packages/**/*.{js,ts,tsx,scss}"
+						"packages/**/*.{js,ts,tsx,scss}",
+						"tests/js/*",
```

Comment:
Is it intentional that you have it running on any change in these folders?  Like if there's a README added or something, have the tests run?

I can't see it being a huge problem, just noting that the assets and packages folders specify filetypes.

Reply:
We could target json, js, tsx ,ts specifically instead. that was the main reason I put the * because I didn't anticipate a readme file being placed here. I will make this change though because it's simple enough :)

Reply:
Thanks for the suggestion @lanej0 fixed in https://github.com/woocommerce/woocommerce/pull/47513/commits/ec2ec0e9453b8eb76b146ab0f9b5b2565c442cc4